app/controllers/nonsipp/otherassetsdisposal/StartReportingAssetsDisposalController.scala

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.nonsipp.otherassetsdisposal
+
+import cats.implicits.toTraverseOps
+import com.google.inject.Inject
+import config.Constants
+import config.Refined.Max5000
+import config.Refined.Max5000.enumerable
+import controllers.PSRController
+import controllers.actions.IdentifyAndRequireData
+import controllers.nonsipp.otherassetsdisposal.StartReportingAssetsDisposalController._
+import forms.RadioListFormProvider
+import models.SchemeId.Srn
+import models.requests.DataRequest
+import models.{Mode, Money, Pagination, SchemeHoldBond}
+import navigation.Navigator
+import navigation.nonsipp.findNextOpenIndex
+import pages.nonsipp.otherassetsdisposal._
+import pages.nonsipp.otherassetsheld._
+import play.api.data.Form
+import play.api.i18n.MessagesApi
+import play.api.mvc.{Action, AnyContent, MessagesControllerComponents, Result}
+import utils.ListUtils._
+import viewmodels.DisplayMessage.{Message, ParagraphMessage}
+import viewmodels.LegendSize
+import viewmodels.implicits._
+import viewmodels.models.{FormPageViewModel, ListRadiosRow, ListRadiosViewModel, PaginatedViewModel}
+import views.html.ListRadiosView
+
+import javax.inject.Named
+
+class StartReportingAssetsDisposalController @Inject()(
```

Comment:
Do we need a test for this? Can be added later on, shouldn't block the merge of this.